INeedCoffee/coffeeShop.rb

```diff
@@ -0,0 +1,25 @@
+require './calculate'
+require './import'
+require './display'
+require './sort'
+
+class CoffeeShop
+  def self.INeedCoffee()
+    userLatitude = Float(ARGV[0])
```

Comment:
If one does not pass the right params the app will crash with a very strange stacktrace:

```
Traceback (most recent call last):
	2: from coffeeShop.rb:25:in `<main>'
	1: from coffeeShop.rb:8:in `INeedCoffee'
coffeeShop.rb:8:in `Float': can't convert nil into Float (TypeError)
```

It will be great to display minimal usage information, like you have in other command line tools like `git`